examples/cli-instrumentation/otel/otel.ts

```diff
@@ -0,0 +1,40 @@
+import { getNodeAutoInstrumentations } from "@opentelemetry/auto-instrumentations-node";
+import { PrometheusExporter } from "@opentelemetry/exporter-prometheus";
+import { OTLPTraceExporter } from "@opentelemetry/exporter-trace-otlp-http";
+import {
+  defaultResource,
+  resourceFromAttributes,
+} from "@opentelemetry/resources";
+import { NodeSDK } from "@opentelemetry/sdk-node";
+import { BatchSpanProcessor } from "@opentelemetry/sdk-trace-node";
+import {
+  ATTR_SERVICE_NAME,
+  ATTR_SERVICE_VERSION,
+} from "@opentelemetry/semantic-conventions";
+
+// Create a resource that identifies your service
+const resource = defaultResource().merge(
+  resourceFromAttributes({
+    [ATTR_SERVICE_NAME]: "apibara",
+    [ATTR_SERVICE_VERSION]: "1.0.0",
+  }),
+);
+
+// By default port 9464 will be exposed on the apibara app
+const prometheusExporter = new PrometheusExporter();
```

Comment:
We should use [`@opentelemetry/exporter-metrics-otlp-grpc`](https://www.npmjs.com/package/@opentelemetry/exporter-metrics-otlp-grpc) and export to the local opentelemetry collector (same as the traces).